jrnl/controller.py

```diff
@@ -123,9 +124,78 @@ def _is_write_mode(args: "Namespace", config: dict, **kwargs) -> bool:
     return write_mode
 
 
+def _read_template_file(template_arg: str, template_path_from_config: str) -> str:
```

Comment:
I'm thinking that this function should be split up into shorter functions and at least some of them should be moved elsewhere. `editor.py` has previously been the only file involved with templates and I think some of these concerns belong there. Ideally, the controller is just telling other classes what to do, rather than doing the work itself.

Though let me know if you're not interested in that work -- we're happy to do it as well.

Reply:
> I'm thinking that this function should be split up into shorter functions and at least some of them should be moved elsewhere.

Seems reasonable enough to me. I don't have a strong opinion on it.

> Though let me know if you're not interested in that work -- we're happy to do it as well.

I'll pass on implementing it myself :) I'm already running a serious open-source time deficit.